### PR TITLE
Few features

### PR DESCRIPTION
**Pull request by archetipico**
Added `pkg`, moved it with `emerge` on top.
Made a proper access feature.
Tested on [Termux](https://termux.com/) too.

### CODE REVIEW

File: updater.sh [hunk 1]
```diff
@@ -1,22 +1,70 @@
...
+    printProgress "deepclean: completed\n"
+
+    exit 1;
+fi
+
+printProgress "Checking for sudo privileges"
+sudo -v >/dev/null 2>&1
+if [[ "$(echo $?)" -eq 0 ]]; then
+    printProgress "Access granted.\n"
+    SUDO="sudo"
```
Comment (MasterCruelty) [inline updater.sh]: This variable isn't used anymore after the definition. @archetipico

**Review by MasterCruelty — APPROVED** [review]
👓 Reviewed.

File: updater.sh [hunk 2]
```diff
@@ -1,22 +1,70 @@
...
+    $PKG --depclean --ask
+    revdep-rebuild
+    printProgress "deepclean: completed\n"
+
+    exit 1;
+fi
+
+printProgress "Checking for sudo privileges"
+sudo -v >/dev/null 2>&1
+if [[ "$(echo $?)" -eq 0 ]]; then
```
[the remark this answers is not in the document]
Comment (archetipico) [reply updater.sh:60]: We should do it

File: updater.sh (hunk 1 above)
Comment (archetipico) [reply updater.sh]: I forgot to remove it 😮

**Review by archetipico — COMMENTED** [review]
Nothing to review, all nice

**Pull request merged**